packages/next/server/dev/hot-middleware

```diff
@@ -0,0 +1,141 @@
+"use strict";
```

Comment:
This should definitely not be here 🤔

Reply:
Why on earth did this happen? The only file i edited was the one with the content changes. Not any of these extras 😕

Reply:
Removing the files that should not be added, seems something with compilation went wrong and you ended up committing those files unintentionally.